test/node.ts

```diff
@@ -0,0 +1,6 @@
+/* eslint-env mocha */
```

Comment:
Consider a different name for this file.

Reply:
The name seems to be the norm for `aegir test`, which I'm borrowing from *star.

If I _just_ rename it without doing stuff to the package.json I get

````
Error: No test files found
Command failed with exit code 1: mocha test/node.{js,cjs,mjs} test/**/*.spec.{js,cjs,mjs} dist/test/node.{js,cjs,mjs} dist/test/**/*.spec.{js,cjs,mjs} --ui bdd --require source-map-support/register --timeout=60000 --bail
error Command failed with exit code 1.
````

I think I can make it work, though.

````
$./node_modules/.bin/mocha dist/test/notnode.js


  0 passing (2ms)
````

Do you have a preferred name? tests.ts ?

Reply:
If there will only be a single file in this folder, maybe `index.ts`?  If you'll mix unit and e2e, then `unit.ts`.